packages/runtime/container-runtime/src/opLifecycle/opSplitter.ts

```diff
@@ -35,7 +54,10 @@ export class OpSplitter {
         if (chunkedContent.chunkId < chunkedContent.totalChunks) {
             // We are processing the op in chunks but haven't reached
             // the last chunk yet in order to reconstruct the original op
-            return message;
+            return {
+                message,
+                state: "Accepted",
+            };
         }
 
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Comment for next line: Not sure if we can do anything here to make our lives easier for the future, but we will eventually move toward binary payloads (and remove overhead of base64). I worry that finding all the places where we rely on string will be hard.